gulpfile.js

```diff
@@ -1,12 +1,12 @@
 /* eslint-disable @typescript-eslint/no-var-requires */
 const sass = require("gulp-sass");
-const importer = require("node-sass-tilde-importer");
 const postcss = require("gulp-postcss");
 const cssnano = require("cssnano");
 const autoprefixer = require("autoprefixer");
 const rename = require("gulp-rename");
+const Fiber = require("fibers");
```

Comment:
Fibers is a libary made to bridge the gap for code written in a  synchronous language and executen in a asynchronous language. This allows us to compile asynchronously, while keeping more or less the performance.